protocol/communities/manager.go

```diff
@@ -308,8 +308,9 @@ type Subscription struct {
 }
 
 type CommunityResponse struct {
-	Community *Community        `json:"community"`
-	Changes   *CommunityChanges `json:"changes"`
+	Community      *Community        `json:"community"`
+	Changes        *CommunityChanges `json:"changes"`
+	RequestsToJoin []*RequestToJoin  `json:"requestsToJoin"`
```

Comment:
sorry, but why do we need it? Can't we use members added/removed from the changes?

Reply:
I guess I didn't think about that, but it doesn't feel right since the member isn't added yet, since it's an admin event.

Also, the ActivityCenter function needs a request ID, so it would be a bit of a mess to reconvert to a request in the messenger IMO